src/mlx/warnings.py

```diff
@@ -148,13 +154,30 @@ def toggle_printout(self, printout):
         '''
         self.printout = printout
 
+    def config_parser(self, config):
+        ''' Parsing configuration dict extracted by previously opened json file
+
+        Args:
+            config (dict): json dump of the configuration
+        '''
+        self.publicCheckers = [SphinxChecker(), DoxyChecker(), JUnitChecker()]
+        # activate checker
+        for checker in self.publicCheckers:
+            try:
+                if bool(config[checker.name]['enabled']):
+                    self.activate_checker(checker)
+                    self.get_checker(checker.name).set_maximum(int(config[checker.name]['max']))
+                    self.get_checker(checker.name).set_minimum(int(config[checker.name]['min']))
+                    print("Config parsing for {name} completed".format(name=checker.name))
+            except KeyError as e:
+                print("Uncomplete config. Missing: {key}".format(key=e))
```

Comment:
Incomplete?

Reply:
does this mean all of the checkers need to be entered in the config file, even if the checker is not enabled? I would say if checker is not in json-file it is disabled. This is better to maintain from user point of view if we later on add checkers, and user doesn't care about new ones.

Reply:
Unit tests tell you you do not need to add all . This will just report if some key inside checker structure is missing, not if it is missing completely. Changed incomplete.

Reply:
:+1:  and i didn't walk through all of the unit test cases :sweat_drops: